src/instruments/src/EFB/Settings/Settings.tsx

```diff
@@ -398,6 +400,21 @@ const SimOptionsPage = () => {
                         />
                     </div>
 
+                    <div className="py-4 flex flex-row justify-between items-center">
+                        <span>
+                            <span className="text-lg text-gray-300">External MCDU Server URL</span>
+                        </span>
+                        <SimpleInput
+                            className="w-30 ml-1.5 px-5 py-1.5 text-lg text-gray-300 rounded-lg bg-navy-light
+                            border-2 border-navy-light focus-within:outline-none focus-within:border-teal-light-contrast text-center disabled"
+                            value={mcduServerUrl}
+                            noLabel
+                            onChange={(event) => {
+                                setMcduServerUrl(event);
```

Comment:
Should we do some basic validation with https://developer.mozilla.org/en-US/docs/Web/API/URL ?

Reply:
`SimpleInput` does not have a good system for this type of input validation and I think creating one would be outside the scope of this PR.